### PR TITLE
[PONG] — Dialogue revisions + additions

### PR DESCRIPTION
**Pull request by ShinodaChan**
Going to work on this one for a little while as a passion project. I really enjoy playing Pong with Monika, but I want to try and branch out the dialogue a bit more while refining what's already there. 

### WORK IN PROGRESS

### CODE REVIEW

File: Monika After Story/game/pong.rpy [hunk 1]
```diff
@@ -731,7 +732,7 @@ label mas_pong_dlg_winner:
 
     #The player has lost 3, 8, 13, ... matches in a row.
     elif loss_streak_counter >= 3 and loss_streak_counter % 5 == 3:
-        m 2eka "Come on, [player], I know you can beat me..."
+        m 2eka "You can do this, [player]..."
```
Comment (scpketer) [inline Monika After Story/game/pong.rpy:735]: "You can do this" doesn't really seem to sound good with an ellipsis at the end, I believe.

File: Monika After Story/game/pong.rpy [hunk 2]
```diff
@@ -766,6 +767,10 @@ label mas_pong_dlg_winner:
             m 3hub "Well played, [player], you're really good!"
             m 1tfu "But so am I, ahaha!"
 
+            if random.randint(1,3) == 1:
+                    m 1eka "Rai is a noob."
```
Comment (scpketer) [inline Monika After Story/game/pong.rpy:771]: FYI this still has odds to roll out as 1, thus matching the condition.

File: Monika After Story/game/pong.rpy [hunk 3]
```diff
@@ -973,12 +978,12 @@ label mas_pong_dlg_loser:
...
         else:
-            m 2hub "Great job, [player]!"
-            m 1kua "Ehehe!"
+            m 2hub "You really are something special, [mas_get_player_nickname()]."
+            m 1kua "Every game with you is a thrill."
 
     #Monika loses five times in a row
     elif win_streak_counter == 5:
-        m 2wud "[player]..."
+        m 2wud "[mas_get_player_nickname()]..."
```
Comment (scpketer) [inline Monika After Story/game/pong.rpy:986]: I think the player's name fits here better than the nickname.

Honestly, I can't see things like 'Love...' or 'My love...' suitable here, you could either exclude these with `exclude_names` keyword argument or let `[player]` be.

Also, the nickname will be in lowercase here, consider using `capitalize` keyword argument.

File: Monika After Story/game/pong.rpy (hunk 2 above)
Comment (Rai99) [inline Monika After Story/game/pong.rpy:771]: Don't look at me with some sad expression about me being a noob. 
```suggestion
                    m 1eua "Rai is a noob."
```

Comment (ShinodaChan) [reply Monika After Story/game/pong.rpy:771]: denied

**Comment by ShinodaChan** [comment]
Iceboxing for now, may come back to this later.

**Pull request closed without merging**